_posts/2020-12-02-welcome-new-hire-getting-started-in-frontend-at-rimdev-updated.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Welcome New Hire: Getting started in Frontend at RIMdev (updated)"
+date: 2020-12-02 08:09:18
+tags:
+- Team
+- RIMdev
+- Frontend
+categories: Team
+twitter_text: "Welcome New Hire: Getting started in #Frontend at RIMdev (update)"
+authors: Kevin Hougasian
+image: https://images.unsplash.com/photo-1524530514346-eab43224489b?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2800&q=80
+image_url: https://unsplash.com/photos/AqrD5QQ0ajM
+image_credit: "@ilumire"
+---
+
+<div class="ui info message">
+This is an updated post of the original <a href="/welcome-new-hire-getting-started-in-frontend-at-rimdev/">here</a>.
+</div>
+
+Whether you're new to the frontend at RIMdev or looking for more about our Frontend team, here's a quick overview of our team, tools, and [culture](https://github.com/ritterim/culture)!
+
+As a whole, RIMdev has always strived to use the technology that best suits the task. This post will give you a glimpse into our environment and tooling. Keep in mind, software changes and so do we.
+
+The majority of our frontend team are Mac users, so this post will seem Mac-biased, but if Windows is where you live, that's just fine! We have several Linux enthusiasts across the team as well, however, we don't support it for business use (that includes Linux subsystem for Windows).
+
+
+## Communicating
+
+Communication is at our core; we can accomplish more as our sum that we can as our parts, so we talk. About projects, goals, vision, and the path to get there. Slack is where we live, comment, post [gifs](https://giphy.com), [cat videos](https://www.youtube.com/watch?v=5dsGWM5XGdg), ya know.
+
+- [Slack](https://slack.com/)! We live here
+- [MS Teams](https://products.office.com/en-us/microsoft-teams/group-chat-software). Chat with non-slackers.
+- Outlook
+
+Looking to get up to speed on either? Here are some quick keyboard references for [Slack](https://slack.com/help/articles/201374536-Slack-keyboard-shortcuts) and [Teams](https://support.microsoft.com/en-us/office/keyboard-shortcuts-for-microsoft-teams-2e8e2a70-e8d8-4a19-949b-4c36dd5292d2.)
+
```

Comment:
Change your avator on slack and teams to be your photo

Reply:
i feel like a dev should be able to figure that out? I would hope anyway 😜